source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -70,6 +70,16 @@ See the table below to see a description of each format:
 
 .. _extended_json_example_section:
 
+.. note::
+
+   The ``$uuid`` Extended JSON type is parsed from a string to a
+   :java-docs:`BsonBinary </apidocs/bson/org/bson/BsonBinary.html#<init>(java.util.UUID)>`
+   object of binary subtype 4. For a further discussion of ``$uuid`` field
+   parsing, see the
+   :spec:`special rules for parsing $uuid fields </extended-json.rst#special-rules-for-parsing-uuid-fields>`
+   section in the extended JSON specification.
```

Comment:
```suggestion
   The ``$uuid`` Extended JSON type is parsed from a String to a
   :java-docs:`BsonBinary </apidocs/bson/org/bson/BsonBinary.html#<init>(java.util.UUID)>`
   object of binary subtype 4. For more information about ``$uuid`` field
   parsing, see the
   :spec:`special rules for parsing $uuid fields </extended-json.rst#special-rules-for-parsing-uuid-fields>`
   section in the extended JSON specification.
```

Reply:
Thanks for the notes. I agree with both (in addition "for more information..." is used often elsewhere in the docs). Committing your suggestion. 

Going forward, I will try to stress simple sentences (and parallel structure), and appropriately capitalize language types.